src/main/java/io/tiledb/java/api/Array.java

```diff
@@ -996,6 +996,88 @@ public ArraySchema getSchema() throws TileDBError {
     return _schema;
   }
 
+  /**
+   * Sets the starting timestamp to use when opening (and reopening) the array. This is an inclusive
+   * bound. The default value is `0`.
+   */
+  public void setOpenTimestampStart(BigInteger timestamp) throws TileDBError {
+    try {
+      ctx.handleError(
+          tiledb.tiledb_array_set_open_timestamp_start(ctx.getCtxp(), getArrayp(), timestamp));
```

Comment:
What happens if someone passes a `timestamp` `BigInteger` that is larger than `UINT64_MAX`? Do we need a range check here -- the timestamp is only valid up to `UINT64_MAX` but IIUC `BigInteger` supports arbitrary precision.

Reply:
Yes that makes sense. However, I noticed that we do not perform such checks throughout the code. Do you think I should open a new PR and add checks in the whole API?

Reply:
Opened ch8569, marking resolved.